modules/ols-activating-token-quota-limits.adoc

```diff
@@ -0,0 +1,72 @@
+// Module included in the following assemblies:
+// * lightspeed-docs-main/configure/ols-configuring-openshift-lightspeed.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ols-activating-token-quota-limits_{context}"]
+= Activating token quota limits
+
+Activate token quota limits for the {ols-long} Service by defining key-value pairs in the `ConfigMap` resource. The {ols-long} pod mounts the `ConfigMap` resource as a volume, enabling access to the file stored within it. The `OLSConfig` Custom Resource (CR) references the `ConfigMap` resource to obtain the quota limit information.
+
+.Prerequisites
+
+* You have installed the the {ols-long} Operator.
+
+* You have configured a large language model provider (LLM).
+
+* A PostgresSQL database is configured and the {ols-long} Service can access the database.
+
+.Procedure
+
+. Open the {ols-long} `OLSconfig` CR file by running the following command:
++
+[source,terminal]
+----
+$ oc edit olsconfig cluster
+----
+
+. Modify the `data` property of the `spec.ols.quotaHandlersConfig` specification to include token quota limit information.
++
+.Example {ols-long} `OLSconfig` file
+[source,yaml]
+----
+apiVersion: v1
+kind: OLSconfig
+metadata:
+  name: quota-limit
+  namespace: openshift-lightspeed
+spec:
+  llm:
+  ols:
+    quota_handlers.conf: 
```

Comment:
Where did this come from? While trying in 0.3.5, I found that only quotaHandlersConfig section was accepted.